tasks/gitlab_permissions.yml

```diff
@@ -0,0 +1,62 @@
+---
+- name: "Set owner of gitlab directory {{ gitlab_install_path }}"
+  file:
+    path: "{{ gitlab_install_path | mandatory }}"
+    owner: git
+    recurse: yes
+
+- name: Place gitlab config
```

Comment:
Why is this in the permissions file? Also why don't you set permissions?

Reply:
move somwhere else